src/k8s-engine/lima.ts

```diff
@@ -455,7 +455,35 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     }
   }
 
-  protected updateConfigPortForwards(config: LimaConfiguration) {
+  protected async checkExpandedLength(proposedPath: string) {
```

Comment:
I think `checkExpandedLength` is a bit obscure. You really need to read the function logic to understand it is about the length of the path after symlinks have been expanded.

I would split this into 2 functions: `EvaluateSymlinks` and `CheckMaxSocketLength` or similar, as these are really unrelated functionalities.